client/utils/validate-token.js

```diff
@@ -17,13 +17,18 @@ const validateToken = async (
   language,
 ) => {
   const url = validateApiUrl(orgSlug);
-  const authToken = cookies.get(`${orgSlug}_auth_token`);
+  const authToken = userData.auth_token
+    ? userData.auth_token
+    : cookies.get(`${orgSlug}_auth_token`);
```

Comment:
what about:

```javascript
// get auth token from redux state, otherwise try getting it from cookies
const authToken = userData.auth_token || cookies.get(`${orgSlug}_auth_token`);
```